From 6f671042d9c3b9b60f515b0a2cc158540e89883a Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 20 Dec 2024 11:07:01 +1100
Subject: [PATCH 4/4] add vt_gen target as a dependency of llvm/lib/Analysis

Various files include `llvm/include/llvm/CodeGenTypes/MachineValueType.h`
which requires `llvm/CodeGen/GenVT.inc`, which is what `vt_gen` depends on.
---
 llvm/lib/Analysis/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/llvm/lib/Analysis/CMakeLists.txt b/llvm/lib/Analysis/CMakeLists.txt
index 5cf3777253fb..be6037515c6d 100644
--- a/llvm/lib/Analysis/CMakeLists.txt
+++ b/llvm/lib/Analysis/CMakeLists.txt
@@ -151,6 +151,7 @@ add_llvm_component_library(LLVMAnalysis
 
   DEPENDS
   intrinsics_gen
+  vt_gen
   ${MLDeps}
 
   LINK_LIBS
